### PR TITLE
PCL SiPixel Alignment: integration of quality checks on payload and DQM module

### PR DESCRIPTION
**Pull request by cerminar**
This PR modifies only the PCL sequences for the alignment of the Pixel large structures.
It integrates the needed checks for the creation of the payload and the DQM module to monitor the movements of the alignables.

This is needed to start testing the new workflow in production environment.

### CODE REVIEW

**Comment by mmusich** [comment]
please test

**Comment by mmusich** [comment]
+1

File: Alignment/MillePedeAlignmentAlgorithm/plugins/BuildFile.xml [hunk 1]
```diff
@@ -2,5 +2,7 @@
 <use name="FWCore/PluginManager"/>
 <use name="FWCore/ParameterSet"/>
 <use name="CondFormats/Common"/>
-<flags EDM_PLUGIN="1"/>
-
+<use name="DQMServices/Core"/>
+<use name="PhysicsTools/UtilAlgos"/>
```
Comment (davidlange6) [inline Alignment/MillePedeAlignmentAlgorithm/plugins/BuildFile.xml]: @cerminar - why is this needed? (I hope it is not)

File: Alignment/MillePedeAlignmentAlgorithm/plugins/MillePedeDQMModule.cc [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+ * @author    Max Stark (max.stark@cern.ch)
+ * @date      Feb 19, 2016
+ */
+
+
+/*** header-file ***/
+#include "Alignment/MillePedeAlignmentAlgorithm/plugins/MillePedeDQMModule.h"
+
+/*** ROOT objects ***/
+#include "TCanvas.h"
```
Comment (davidlange6) [inline Alignment/MillePedeAlignmentAlgorithm/plugins/MillePedeDQMModule.cc]: @cerminar - why is this needed?

**Comment by mmusich** [comment]
please test

**Comment by mmusich** [comment]
+1
provided jenkins agrees

**Comment by davidlange6** [comment]
+1
